.github/scripts/stub.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+import { existsSync, mkdirSync, readFileSync } from "fs";
+import { basename, dirname, join, resolve } from "path";
+import child_process from "child_process";
+import XmlReader from "xml-reader";
+import xmlQuery from "xml-query";
+
+const DSP_DIR = getDSPFolder();
+const BEPINEX_DIR = join(DSP_DIR, "BepInEx\\core\\");
+const DSP_ASSEMBLY_DIR = join(DSP_DIR, "DSPGAME_Data\\Managed\\");
+
+const refFile = readReferenceFile();
+
+function main() {
+  if (!existsSync("Libs")) {
+    mkdirSync("Libs");
+  }
+  publicizeAndStubAssemblies(getReferencePaths());
+}
+
+function getDSPFolder() {
+  const targetsFile = "DevEnv.targets";
+
+  var nebulaPath =
+    "C:\\Program Files (x86)\\Steam\\steamapps\\common\\Dyson Sphere Program";
+
+  if (existsSync(targetsFile)) {
+    const xml = XmlReader.parseSync(readFileSync(targetsFile, "utf-8"));
+    const tmpPath = xmlQuery(xml).find("DSPGameDir").text();
+    if (existsSync(tmpPath)) {
+      nebulaPath = tmpPath;
+    }
+  }
+
+  return join(nebulaPath);
```

Comment:
Why do you do a `path.join` with only one value ?

Reply:
> Why do you do a `path.join` with only one value ?

it's the same purpose as EnsureTrailingSlash in the directory.build.props, just to make sure it ends in a /... I think it works like that at least 🤔